src/models/loginService.js

```diff
@@ -14,6 +14,9 @@ export async function loginUser(credentials) {
   } catch (error) {
     // Not 2xx responses.
     console.error('Failed to login', error.response);
+    if (error.response.status !== 500){
```

Comment:
I don't you need to distinguish what error status it is, because `error.response.data.error` will contain a meaningful error message from the backend. The default `'An error occured'` is fine because it will only be used if the  `error.response.data.error` doesn't exist(null, undefined etc).